### DISCUSSION TITLE
The "-e" option does not exist. when i tried to create a crud

### DISCUSSION THREAD

**Discussion by Crowzed** [Laravel-Backpack/community-forum#1029] [Q&A (Help)]
![image](https://github.com/Laravel-Backpack/community-forum/assets/110807022/b11698d6-f58a-41ca-a19d-36e68135eb56)
![image](https://github.com/Laravel-Backpack/community-forum/assets/110807022/2d4f609d-8def-4e0a-8a77-538987a0c49e)

**Comment by jcastroa87** [comment]
Hello @Crowzed 

Can you please show full screenshot or write the command you use here.

Did you fill all the steps basset need?
1. Set APP_URL
2. Run your server previously
3. Set properly the public disk
4. Run the storage symlink:
```
php artisan storage:link
```

Let me know further information.

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
